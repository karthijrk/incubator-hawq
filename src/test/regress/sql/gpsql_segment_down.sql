SET gp_test_failed_segmentid_number = 1;
-- SEGMENT ID 1 FAILED
SET gp_test_failed_segmentid_start = 1;
