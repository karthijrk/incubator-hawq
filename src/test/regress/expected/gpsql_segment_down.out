SET gp_test_failed_segmentid_number = 1;
ERROR:  segment alive information changed, cleanup the gangs. (postgres.c:1800)
-- SEGMENT ID 1 FAILED
SET gp_test_failed_segmentid_start = 1;
ERROR:  segment alive information changed, cleanup the gangs. (postgres.c:1800)
